client/state/preferences/README.md

```diff
@@ -45,5 +45,6 @@ To add a new preference key, you only need to add a key and defaut value in `DEF
 ### Actions
 
 - **fetchPreferences()** - populate the tree
-- **setPreference( key, value )** - set preference and persist all preferences in the endpoint
+- **savePreference( key, value )** - set preference and persist all preferences in the endpoint
```

Comment:
How about `setPreference` and `setLocalPreference` instead to clarify that the latter is not persisted in the API.

For what use cases do we need to store preferences for a shorter period of time and how long should they be stored? Currently preferences have the potential to be persisted to browser storage, so they'll last for more than a page view.